eoldates/testdata/happy/eoldates.json

```diff
@@ -0,0 +1,26 @@
+{
+  "alpine-linux": [
+    {
+      "name": "3.22",
+      "eolFrom": "2027-05-01"
+    }
+  ],
+  "oracle-linux": [
+    {
+      "name": "9",
+      "eolFrom": "2032-06-30",
+      "eoesFrom": "2035-06-30"
+    }
+  ],
+  "ubuntu": [
+    {
+      "name": "25.04",
+      "eolFrom": "2026-01-17"
+    },
+    {
+      "name": "22.04",
+      "eolFrom": "2027-04-01",
+      "eoesFrom": "2032-04-09"
```

Comment:
I was thinking about ubuntu ESM:
where we should handle common and ESM versions.

I decided to use our logic for `vuln-list-update` (keep data as in source). We will handle (split) ESM version in trivy-db.